react-js/react-part-2.md

```diff
@@ -0,0 +1,176 @@
+# React Part 2
+
+## Projected Time
+
+1.5 hours
+- Lesson: 30min
+- Guided Practice: 30min
+- Independent Practice: 30min
+
+### Prerequisites
+
+- [React](react.md)
+- [JSX](react.md)
+- [Props](react.md)
+
+### Motivation
+
+In introduction to [React](/react.md) we discussed about how React makes websites interactive and reduces jQuery code. Now the problem arises if we want to store some information which intent to change and due to change of that information the UI changes i.e the page is re-rendered. Thus, component state is used for such cases.
+
+### Objectives
+
+- Component state
+- State vs Props
+- State are Immutable
+
+### Lesson
+
+[Slides](https://docs.google.com/presentation/d/1Trj_7vWYe79_mDboDxN9rdT94yjY5Dt55xtU1k3dUsw/edit?usp=sharing)
```

Comment:
Isn't the presentation public ? It's showing me access denied.
@ashishnagpal2498 And please specify what the slide is about.

Reply:
Yes , it wasn't. I just corrected it, you can view it now. Also I have changed the owner.

Reply:
I need edit access to review the presentation. Until you fix other issues.

Reply:
Kindly provide me your email @Kundan28

Reply:
I already requested, you might have received an email.

Reply:
No , I haven't received any mail. May be because I have transferred the ownership. Provide your email so that I can give you the access.